bin/truffleruby.sh

```diff
@@ -92,14 +92,16 @@ source_truffle="$root_parent/graal/truffle/mxbuild"
 if [ -f "$binary_truffle/dists/truffle-api.jar" ]; then # Binary Truffle suite
     truffle="$binary_truffle"
     graal_sdk="$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/graal-sdk.jar"
+    launcher_common="$(dirname "$binary_truffle")/mx.imports/binary/sdk/mxbuild/dists/launcher-common.jar"
 elif [ -f "$source_truffle/dists/truffle-api.jar" ]; then # Source Truffle suite
     truffle="$source_truffle"
     graal_sdk="$root_parent/graal/sdk/mxbuild/dists/graal-sdk.jar"
+    launcher_common="$root_parent/graal/sdk/mxbuild/dists/launcher-common.jar"
 else
     echo "Could not find Truffle jars" 1>&2
     exit 1
 fi
-java_args+=("-Xbootclasspath/a:$truffle/dists/truffle-api.jar:$graal_sdk")
+java_args+=("-Xbootclasspath/a:$truffle/dists/truffle-api.jar:$graal_sdk:$launcher_common")
```

Comment:
Does `launcher-common.jar` need to be on the bootclasspath or is it enough on the classpath?

Reply:
I think bootpath. I am not 100% sure, but I don't think it makes sense to go and find out. This is a developer launcher which is not distributed. I don't think there is any practical impact of the placement.

Reply:
Right, fair enough and the rest of Graal SDK is there too.
Note that this launcher is notably used for the testing distribution and distributions without a SVM image launcher.